src/ops/fuelup_check.rs

```diff
@@ -53,95 +49,94 @@ fn format_version_comparison(current_version: &Version, latest_version: &Version
                 colored_bold(Color::Green, "(recommended)")
             )
         }
-    }
+        Equal => {
+            // Only show up-to-date message if verbose is true
+            if verbose {
+                format!(
+                    "{} : {current_version}",
+                    colored_bold(Color::Green, "Up to date")
+                )
+            } else {
+                return None;
+            }
+        }
+    };
+    Some(s)
 }
 
-fn check_plugin(plugin_executable: &Path, plugin: &str, latest_version: &Version) {
-    let version_or_err = match get_bin_version(plugin_executable) {
-        Ok(version) => format_version_comparison(&version, latest_version),
-        Err(err) => err.to_string(),
-    };
-    info!("{:>4}- {} - {}", "", plugin, version_or_err);
+fn check_plugin(plugin_executable: &Path, plugin: &str, latest_version: &Version, verbose: bool) {
+    if let Some(version_or_err) = match get_bin_version(plugin_executable) {
+        Ok(version) => format_version_comparison(&version, latest_version, verbose),
+        Err(err) => Some(err.to_string()),
+    } {
+        info!("{:>4}- {} - {}", "", plugin, version_or_err);
+    }
 }
 
-fn check_fuelup() -> Result<()> {
+fn check_fuelup(verbose: bool) -> Result<()> {
     let fuelup_version: Version = Version::parse(clap::crate_version!())?;
-
-    if let Ok(fuelup_download_cfg) = DownloadCfg::new(
-        component::FUELUP,
-        TargetTriple::from_component(component::FUELUP)?,
-        None,
-    ) {
-        info!(
-            "{} - {}",
-            bold(component::FUELUP),
-            format_version_comparison(&fuelup_version, &fuelup_download_cfg.version)
-        );
+    if let Ok(latest) = get_latest_version(component::FUELUP) {
+        if let Some(text) = format_version_comparison(&fuelup_version, &latest, verbose) {
+            info!("{} - {}", bold(component::FUELUP), text);
+        }
     } else {
-        error!("Failed to create DownloadCfg for component 'fuelup'; skipping check for 'fuelup'");
+        error!("Failed to get latest version for component 'fuelup'; skipping check for 'fuelup'");
     }
     Ok(())
 }
 
 fn check_toolchain(toolchain: &str, verbose: bool) -> Result<()> {
     let description = DistToolchainDescription::from_str(toolchain)?;
-
     let dist_channel = Channel::from_dist_channel(&description)?;
     let latest_package_versions = collect_package_versions(dist_channel);
-
     let toolchain = Toolchain::new(toolchain)?;
+    info!("{}: {}", bold("Toolchain: "), &toolchain.name);
```

Comment:
```suggestion
    info!("{}{}", bold("Toolchain: "), &toolchain.name);
```

![image](https://github.com/FuelLabs/fuelup/assets/47993817/6d9fe984-0da4-42e7-a016-f6421a04798c)

I get the need for this label, because with all of the components listed below it it's hard to see that that's the toolchain name. It's a lot more readable without the download bar! 👍 

Since we have the `-v` added back, I wonder if we should go further with condensing the output in non-verbose mode. It could be like this:

```
latest-aarch64-apple-darwin - Up to date
nightly-aarch64-apple-darwin - Update available (12 updates)
```

Reply:
I like this idea :)

Reply:
putting into draft mode until I complete this

Reply:
done in [05a3363](https://github.com/FuelLabs/fuelup/pull/631/commits/05a336343b91089d0b0156d0567c99053c284121)